app/assets/stylesheets/_dashboard.less

```diff
@@ -2,79 +2,166 @@
   margin-top: 0px;
 }
 
-.dataset-thumbnail {
-  img {
-    opacity: 0.8;
-    position: relative;
-    object-fit: cover;
-    height: 200px;
-    width: 100%;
-    display: block;
-  }
-}
+.gallery-dataset-col{
+  paddingBottom: 16px;
 
-.dataset-thumbnail-buttons {
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  width: 100%;
-  height: 200px;
 
-  a {
-    box-shadow: white 0px 0px 4px;
-    width: 40px;
-    height: 40px;
-    border-radius: 10px;
-    background: rgba(0, 0, 0, 0.8);
-    display: inline-block;
+  .spotlight-item-card {
+    height: 300px;
```

Comment:
Because if the width is below 600px the thumbnails is displayed in full width. If you subtract padding and all you get a max width of 512px that is about 500px.